src/spikeinterface/core/waveforms_extractor_backwards_compatibility.py

```diff
@@ -614,13 +616,12 @@ def make_ext_params_up_to_date(ext, old_params, new_params):
     old_name = ext.extension_name
     updated_params = old_params.copy()
     for p, values in old_params.items():
-        if isinstance(values, dict):
+        if p not in new_params:
+            warnings.warn(f"Removing legacy param {p} from {old_name} extension")
```

Comment:
```suggestion
            warnings.warn(f"Removing legacy parameter {p} from {old_name} extension")
```

should we be more explicit for the end user. We call it param in the code base, but the end user would understand parameter better, no?